yaksh/templates/yaksh/course_detail.html

```diff
@@ -151,4 +182,8 @@
         </div>
     </div>
 </div>
+<!-- Dialog to display error message -->
+<div id="dialog" title="Alert">
```

Comment:
Hey, so on testing, it seems that I can't get rid of the alert box until I press esc key. I am not sure what's causing that though.